floats/floats.go

```diff
@@ -567,6 +568,19 @@ func Norm(s []float64, L float64) float64 {
 	return math.Pow(norm, 1/L)
 }
 
+// ParseWithNA converts the string s to a float64 in v.
+// If s equals na, w is returned as 0, otherwise 1.
```

Comment:
Can we somehow use the phrase "not available" into the documentation? I feel the purpose isn't very clear, especially for non-native speakers.